### PR TITLE
Collections.java replace by apache.commons.collections4

### PR DESCRIPTION
**Pull request by rjbhewei**
code clean

### CODE REVIEW

**Comment by rjbhewei** [comment]
I signed it

File: util/src/main/java/io/kubernetes/client/informer/SharedInformerFactory.java [hunk 1]
```diff
@@ -1,20 +1,16 @@
 package io.kubernetes.client.informer;
 
+import java.lang.reflect.Type;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.concurrent.*;
```
Comment (yue9944882) [inline util/src/main/java/io/kubernetes/client/informer/SharedInformerFactory.java]: flatten the wildcard imports (this comment applies throughout this pull)

File: util/src/main/java/io/kubernetes/client/informer/SharedInformerFactory.java [hunk 2]
```diff
@@ -4,17 +4,12 @@
 import com.squareup.okhttp.Call;
 import io.kubernetes.client.*;
 import io.kubernetes.client.informer.impl.DefaultSharedIndexInformer;
-import io.kubernetes.client.util.CallGenerator;
-import io.kubernetes.client.util.CallGeneratorParams;
-import io.kubernetes.client.util.Watch;
-import io.kubernetes.client.util.common.Collections;
+import io.kubernetes.client.util.*;
```
Comment (yue9944882) [inline util/src/main/java/io/kubernetes/client/informer/SharedInformerFactory.java]: plz unwound the wildcards

**Comment by rjbhewei** [comment]
recheck

File: util/src/main/java/io/kubernetes/client/informer/SharedInformerFactory.java (hunk 2 above)
Comment (yue9944882) [reply util/src/main/java/io/kubernetes/client/informer/SharedInformerFactory.java]: the wildcard imports still exists, am i missing anything?

File: util/src/main/java/io/kubernetes/client/informer/SharedInformerFactory.java [hunk 3]
```diff
@@ -4,17 +4,12 @@
...
 import java.lang.reflect.Type;
 import java.util.HashMap;
 import java.util.Map;
-import java.util.concurrent.ExecutorService;
-import java.util.concurrent.Executors;
-import java.util.concurrent.Future;
-import java.util.concurrent.TimeUnit;
+import java.util.concurrent.*;
```
Comment (yue9944882) [inline util/src/main/java/io/kubernetes/client/informer/SharedInformerFactory.java]: > the wildcard imports still exists, am i missing anything?

same here

File: util/src/main/java/io/kubernetes/client/informer/cache/DeltaFIFO.java [hunk 4]
```diff
@@ -1,12 +1,10 @@
 package io.kubernetes.client.informer.cache;
 
-import io.kubernetes.client.util.common.Collections;
 import java.util.*;
-import java.util.concurrent.locks.Condition;
-import java.util.concurrent.locks.ReadWriteLock;
-import java.util.concurrent.locks.ReentrantReadWriteLock;
+import java.util.concurrent.locks.*;
```
Comment (yue9944882) [inline util/src/main/java/io/kubernetes/client/informer/cache/DeltaFIFO.java]: > the wildcard imports still exists, am i missing anything?

ditto

File: util/src/main/java/io/kubernetes/client/informer/impl/DefaultSharedIndexInformer.java [hunk 5]
```diff
@@ -1,13 +1,9 @@
 package io.kubernetes.client.informer.impl;
 
-import io.kubernetes.client.informer.ListerWatcher;
-import io.kubernetes.client.informer.ResourceEventHandler;
-import io.kubernetes.client.informer.SharedIndexInformer;
+import io.kubernetes.client.informer.*;
```
Comment (yue9944882) [inline util/src/main/java/io/kubernetes/client/informer/impl/DefaultSharedIndexInformer.java]: > the wildcard imports still exists, am i missing anything?

ditto

File: util/src/main/java/io/kubernetes/client/informer/impl/DefaultSharedIndexInformer.java [hunk 6]
```diff
@@ -1,13 +1,9 @@
...
 import io.kubernetes.client.informer.cache.*;
-import io.kubernetes.client.util.common.Collections;
-import java.util.Deque;
-import java.util.List;
-import java.util.Map;
+import java.util.*;
```
Comment (yue9944882) [inline util/src/main/java/io/kubernetes/client/informer/impl/DefaultSharedIndexInformer.java]: > the wildcard imports still exists, am i missing anything?

ditto

File: util/src/main/java/io/kubernetes/client/informer/impl/DefaultSharedIndexInformer.java [hunk 7]
```diff
@@ -1,13 +1,9 @@
 package io.kubernetes.client.informer.impl;
 
-import io.kubernetes.client.informer.ListerWatcher;
-import io.kubernetes.client.informer.ResourceEventHandler;
-import io.kubernetes.client.informer.SharedIndexInformer;
+import io.kubernetes.client.informer.*;
 import io.kubernetes.client.informer.cache.*;
```
Comment (yue9944882) [inline util/src/main/java/io/kubernetes/client/informer/impl/DefaultSharedIndexInformer.java]: > io.kubernetes.client.informer.cache.*;

we should have expand this too, can you unpack it?

File: util/src/test/java/io/kubernetes/client/CopyTest.java [hunk 8]
```diff
@@ -34,25 +34,28 @@
 
 /** Tests for the Copy helper class */
 public class CopyTest {
-  private String namespace;
-  private String podName;
-  private String[] cmd;
+
+  private String namespace, podName;
```
Comment (yue9944882) [inline util/src/test/java/io/kubernetes/client/CopyTest.java]: the diff from this file doesnt seem related w/ the point of this pull? let's keep a minimum change i.e. revert the diff

**Pull request closed without merging**